builder/comp-builder.nix

```diff
@@ -42,9 +43,9 @@ let self =
 
 , enableStatic ? component.enableStatic
 , enableShared ? ghc.enableShared && component.enableShared && !haskellLib.isCrossHost
-               # on x86 we'll use shared libraries, even with musl m(
-               # ghc's internal linker seems to be broken on x86.
-               && !(stdenv.hostPlatform.isMusl && !stdenv.hostPlatform.isx86)
+               # We'll include shared libraries for musl.
+               # Without them TH currently fails.
+               # TODO consider adding `&& !stdenv.hostPlatform.isMusl`.
```

Comment:
I don't understand this comment?

Reply:
Nor do I.  I have removed this comment.

I still am not sure what the purpose of `&& !(stdenv.hostPlatform.isMusl && !stdenv.hostPlatform.isx86)` was, but non of the tests seem to depend on it.  Do you remember what it was for?